packages/common/src/types/certificate.ts

```diff
@@ -420,3 +420,12 @@ export interface IcfTitles {
     common: string[]
   }
 }
+
+export enum CertificateSignStatus {
+  INITIAL = '',
+  UNKNOWN = 'OKAND',
+  PROCESSING = 'BEARBETAR',
```

Comment:
Hur kommer det sig att vi blandar svenska och engelska enumar här?

Reply:
Good question. I believe we need to have this translation, currently this code is sharing its backend endpoint with the old client, so the values from the backend are in Swedish while the codebase for this project is English. Could be that the translation is perhaps better suited as a backend task.

Should/could we write a translation endpoint that gives us English values? @mww-sopra-steria

Reply:
The enums are part of legacy code and I wouldn't change it as it would require changes in both backend and current angularjs client.

Reply:
Alright, I would like to keep the value translated in this enum then, since I believe it would be much worse if we have Swedish words randomly showing up in other contexts where `CertificateSignStatus` is used. Keeping the Swedish to a minimum :)